openfecli/commands/plan_rbfe_network.py

```diff
@@ -142,27 +139,29 @@ def plan_rbfe_network(
         cofactors = []
     write("\t\tCofactors: " + str(cofactors))
 
-    solvent = SolventComponent()
+    yaml_options = YAML_OPTIONS.get(yaml_settings)
```

Comment:
Would it make sense for the other components to also be defined in the YAML?

The use case I'm thinking is that you might want to rely on your yaml as "the truth of everything you passed through", so that might include some paths to various files you want to use. That way you can always go back and say "o ok, this was the inputs we passed through".

Reply:
It also helps the folks that are somewhere in the "half programmatic" span of things - where they want to script something, but are only confident in writing out yaml files.